contracts/registry/src/registry.rs

```diff
@@ -304,9 +304,40 @@ impl SBTRegistry for Contract {
     #[payable]
     fn sbt_recover(&mut self, from: AccountId, to: AccountId) {
         let issuer = env::predecessor_account_id();
-        self.assert_issuer(&issuer);
+        let issuer_id = self.assert_issuer(&issuer);
         self.assert_not_banned(&from);
         self.assert_not_banned(&to);
+
+        //todo: check if enough near was provided to cover the storage
+        let storage_deposit = env::attached_deposit();
+        require!(
+            storage_deposit >= 6 * MILI_NEAR,
```

Comment:
what's the required storage for it?

Reply:
storage deposit is only required if we create a new storage. Her we don't create a new storage - we re-assign tokens.

Reply:
Makes sense. I was confused by the comments on top of the function that suggested we need the storage deposit